src/ArgumentAccessors.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Spatie\PhpUnitWatcher;
+
+trait ArgumentAccessors
+{
+    public function setTestFile($testFile)
```

Comment:
This can actually be a directory or a file. Maybe a better name would be `setTestPath`